functions.php

```diff
@@ -7,7 +7,7 @@ function connectDb(string $host, string $user, string $password, string $dbName)
     return $pdo;
 }
 
-function getBottles($pdo)
+function getBottles($pdo): array
 {
```

Comment:
the opening curly bracket should be on the same line as the function name because this isn't in a class